draft-ietf-quic-recovery.md

```diff
@@ -634,32 +650,48 @@ are detected lost.
 ~~~
    OnPacketsLost(lost_packets):
      largest_lost_packet = lost_packets.last()
+     // Start a new recovery epoch if the lost packet is larger
+     // than the end of the previous recovery epoch.
      if (end_of_recovery < largest_lost_packet.packet_number):
        end_of_recovery = largest_sent_packet
        congestion_window *= kLossReductionFactor
+       congestion_window = max(congestion_window, kMinimumWindow)
        ssthresh = congestion_window
 ~~~
 
+## On Retransmission Timeout Verified
+
+QUIC decreases the congestion window to the minimum value once the
+retransmission timeout has been confirmed to not be spurious when
+the first post-RTO acknowledgement is processed.
+
+~~~
+   OnRetransmissionTimeoutVerified()
+     congestion_window = kMinimumWindow
+~~
+
 ## Pacing Packets
 
 QUIC sends a packet if there is available congestion window and
 sending the packet does not exceed the pacing rate.
 
-TimeToSend returns infinite if the congestion controller is congestion
-window limited, a time in the past if the packet can be sent
-immediately, and a time in the future if sending is pacing limited.
+TimeToSend returns infinite if the congestion controller is
+congestion window limited, a time in the past if the packet can be
+sent immediately, and a time in the future if sending is pacing
+limited.
 
 ~~~
    TimeToSend(packet_size):
      if (bytes_in_flight + packet_size > congestion_window)
        return infinite
+     pacing_coefficient = 1.25
+     if (congestion_window < ssthresh)
+       pacing_coefficient = 2
```

Comment:
I don't think we need a pacing coeff of 2 here. I never understood why FQ did this, but I don't think we need to have this in the doc. In general I'm less certain that we want to describe the pacing impl as part of the congestion controller... I'm comfortable with definiing the TimeToSend interface without implementing it here. For now, I would at least remove the use of pacing_coeff here.